cordapp-cpk2/src/test/kotlin/net/corda/plugins/cpk2/CordappGradleConfigurationsTest.kt

```diff
@@ -74,40 +74,30 @@ class CordappGradleConfigurationsTest {
 
     @Test
     fun testApiIncluded() {
-        assertThat(testProject.output)
-            .contains("CorDapp library dependency: javax.annotation-api-1.3.2.jar")
```

Comment:
This information is still valid for CPKv2, even if the particular CPKv1 task that used to log it no longer exists. Is there anywhere else you could log this now? (Perhaps not as part of _this_ PR, though.)

Reply:
Thank you Chris. I'll bring back this logging in a future PR.